demos/ultralytics/README.md

```
@@ -0,0 +1,32 @@
+
+2. Copy a video to the `src` folder.
+
+3. Run the demo:
+
+   ```bash
+   python demo.py video.mp4
+   ```
+
+### Options
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)